.gitforbidden

```diff
@@ -0,0 +1,13 @@
+# WARNING: PLEASE KEEP ONE EMPTY STRING AT THE END OF THIS FILE
+# Description:
+# detect beginning of RSA private key
+(\\-){5}BEGIN\\sRSA\\sPRIVATE\\sKEY(\\-){5}
+# Description:
+# detect login/password/token/etc. hardcoded values
+([Ll]ogin|[Pp]assword|[Pp]ass|[Tt]oken|[Uu]ser|[Uu]sername|[sS]ecret|[Pp]rivate\s*):
+# Description:
+# detect IPv4 addresses, but only with at least 2 digits in the first octet (for ignoring versions of libraries)
```

Comment:
can we detect IP blocks as ip/mask instead of assumption that ip can not be a single digit in first octet (because it can)?

Reply:
done